.github/workflows/release.yml

```diff
@@ -26,11 +26,28 @@ jobs:
         uses: snok/install-poetry@v1.3.3
       - name: configure pypi credentials
         run: poetry config pypi-token.pypi "${{ secrets.PYPI_API_KEY }}"
-      - name: Publish package
+      - name: Publish core package
         run: poetry publish --build
       - name: install dependencies
         run: poetry install
       - name: update metadata
         env:
           DATABASE_URL: ${{ secrets.OPENSTATES_DATABASE_URL }}
         run: poetry run os-initdb
+      # Add steps for openstates-metadata package
+      - name: Create directory and copy openstates-metadata
+        run: |
+          mkdir -p /openstates-core-metadata/openstates-metadata
+          cp -r openstates/metadata /openstates-core-metadata/openstates-metadata
+      - name: Change directory to openstates-metadata
```

Comment:
Ignore me if I'm totally misreading this, but seems like this would create this directory structure:

```
/openstates-core-metadata
|_
    | ./openstates-metadata
    |_ 
        | ./_creation
        | ./data
        | ./tests
        | __init__.py
        | models.py
        | pyproject.toml.sample
        | .python-version.sample
```

I would have guessed we want

```
/openstates-core-metadata
|_
    | ./_creation
    | ./openstates-metadata [was ./data]
    | ./tests
    | __init__.py
    | models.py
    | pyproject.toml.sample
    | .python-version.sample
```

could totally be wrong but that was my reading of this

Reply:
This is a working local copy. I think this is right. 
![image](https://github.com/user-attachments/assets/21db04c3-50d9-4c2b-ac18-bed641e5bb74)